Scroller/EditableWrapper.js

```diff
@@ -95,27 +104,42 @@ const EditableWrapper = (props) => {
 	}, [dataSize]);
 
 	const startEditing = useCallback((item) => {
-		// FIXME: Need to figure out the right element
-		if (item.classList.contains('spottable')) {
+		if (item.dataset.index) {
+			Spotlight.set(mutableRef.current.spotlightId, {restrict: 'self-only'});
+
 			item.classList.add(css.selected);
+			item.classList.add(customCss.selected);
```

Comment:
I hope `customCss.selected` can be removed by the polyfill of public classes.

Reply:
As we discussed, 
item.classList.add("AAA BBB") return error, so We cannot remove "item.classList.add(customCss.selected);"